### ISSUE TITLE
19대 총 성적표 기획 의견

### ISSUE THREAD

**Issue by hoony-o-1** [codeforseoul/congress-report#13]
다들 연말/연초라 바쁘신 것 같아요! :+1: 힘 내시길 바랍니다!! ㅜㅜㅋㅋ

우선 19대 총 성적표 관련해서 전달드릴 것이 있어서요. 19대 총 성적표에는 아무래도 웹기획과 시각화 작업이 들어갈텐데 관련한 부분에서 능력자 분의 도움을 얻기위해 소원영 님이라는 분을 초대했어요 !:) 슬랙에서 @wonyoung 아이디를 쓰시는 분입니다.

그리고 어제 핵나잇 때 다들 못오셔서 19대 성적표 기획에 대해 이야기를 못나눴네요. 가능하면 다같이 모여서 행아웃을 하면 좋겠는데 원영님도 미국에 계시고 시간 맞추기가 쉽지 않을 것 같네요. 사실 총 성적표 구현은 데이터쪽보다 웹쪽 이슈라 실제 작업량은 웹이 많을텐데 기획 의견을 다같이 나눠보면 좋을 것 같아서요.

@appetizermonster @shindoh @DongGeun-YI 총 성적표 시각화 작업 관련해서 아이디어 있으시면 같이 나눠주세요! :+1:

**Comment by hoony-o-1** [comment]
국회의원별 법안 발의수 비교, 발의된 법안 현재 상태, 본회의/상임위 출석률 상위/하위 n명

**Comment by hoony-o-1** [comment]
참고
1. [OpenCongress](https://www.opencongress.org/people/show/400440_Don_Young) - Sunlight Foundation
2. [OpenState](http://openstates.org/sd/legislators/SDL000247/anderson-david-l/) - Sunlight Foundation
3. [GovTrack.us](https://www.govtrack.us/congress/members/lisa_murkowski/300075)
4. [19대 의정활동 종합평가](http://the300.mt.co.kr/newsView.html?no=2016010516057627018) - the300(머니투데이)

**Comment by appetizermonster** [comment]
개인적인 생각으로는 머니투데이 자료만큼 점수화를 시키기는 너무 어려울 것 같아서 fact를 잘 나열해서 보여주는 정도면 좋을 것 같습니다.

**Comment by hoony-o-1** [comment]
네 알겠습니다 :+1: 일단 샘플로 데이터 몇개만 추출해보고 해당 데이터를 기반으로 웹기획은 원영님과 함께 해볼게요! 기본적으로 웹 기획은 다음 주 초까지는 완성해보도록 할게요. 월요일에 동동님도 만나구요.

**Comment by shindoh** [comment]
지난번에 말한 쟁점법안이 
http://watch.peoplepower21.org/New/monitor_main_vote_result.php
이거 맞나요?

**Comment by hoony-o-1** [comment]
네 맞아요 :) 참여연대 분이 주요표결 보고서를 공유해주셨는데 공유드릴게요

**Comment by hoony-o-1** [comment]
https://ksj.hackpad.com/--rNA6LRzuXX8

원영님께서 현재 19대 성적표 기획안 & 스토리보드를 구상하면서 디자인 구성을 시작하고 계세요. 의견 있으면 피드백 부탁드려요!!
